README.md

```diff
@@ -12,8 +12,8 @@ Properties
 
 -   **intervals**:
     -   **interval**: After a signal, if another one does not enter the block for this amount of time, emit a timeout signal.
-    -   **repeatable**: If true, a timeout signal is emitted every interval without another input signal instead of just once.
--   **group_by**: Expression proprety. The value by which signals are grouped. Output signals will have *group* set to this value.
+    -   **repeatable**: If `True`, a timeout signal is emitted every interval without another input signal instead of just once.
+-   **group_by**: The value by which signals are grouped. Output signals will have `group` set to this value.
 
 
 Dependencies
```

Comment:
The group by mixin is still used, it's just provided by the nio framework instead of needing to add the mixins as a submodule. So yes, we should still be documenting it's use in blocks that use it.